client/packages/programs/src/JsonForms/components/Search/Search.tsx

```diff
@@ -3,38 +3,40 @@ import { ControlProps, rankWith, uiTypeIs } from '@jsonforms/core';
 import { withJsonFormsControlProps } from '@jsonforms/react';
 import { z } from 'zod';
 import { useZodOptionsValidation } from '../../common/hooks/useZodOptionsValidation';
+import { Typography } from '@openmsupply-client/common';
 import { SearchWithUserSource } from './SearchWithUserSource';
 import { SearchWithDocumentSource } from './SearchWithDocumentSource';
-import { QueryValues } from './useSearchQueries';
 import { SearchWithPatientContactSource } from './SearchWithPatientContactSource';
 
+/**
+ * Specifies the search options when doing a user input type search
+ */
```

Comment:
This is for a patient search only right? Maybe a few more details would be helpful as well:

```
Use the control's scope object (as specified in the control `scope`) to perform a patient search.
For example, if a control is used on a person object and the user edits the person's first name a search for patients with the same first name is done.
The patient query is done using only the fields specified in the `searchFields` option.
When a found patient is select the patient's fields are stored in the control's data object.
The `saveFields` option can be used to define which properties can be saved.
```

EDIT: just realized we already talked about it below :-) I will write an issue for it...